docs: reformat the documentation (#396)
ansys/ansys-sphinx-theme: doc/source/examples.rst

Examples
########

> This show how the different extensions and documentation components are rendered in the ``ansys-sphinx-theme``.

Reviewer: ```suggestion
This section provides examples that show how the Ansys Sphinx Theme renders different extensions and documentation components.
```
Use the "p;roduct" name rather than the package name for better readability.

Sphinx extensions
=================

> These examples show how the different sphinx extensions are rendered in the ``ansys-sphinx-theme``.

Reviewer: ```suggestion
These examples show how the Ansys Sphinx Theme renders different Sphinx extensions.
```

.. grid:: 2
   :gutter: 2 2 3 4
   :padding: 2 2 3 4

   .. grid-item-card:: :octicon:`note` Sphinx design
      :link: sphinx-design
      :link-type: ref

>       Examples of the sphinx design rendered using the `ansys-sphinx-theme`.

Reviewer: ```suggestion
      Examples of how the Sphinx design is rendered.
```

   .. grid-item-card:: :octicon:`book` Jupyter notebook - nbsphinx
      :link: nbsphinx
      :link-type: ref

>       Examples of notebooks rendered using the `ansys-sphinx-theme`.

Reviewer: ```suggestion
      Examples of how notebooks are rendered.
```

   .. grid-item-card:: :octicon:`link-external` Sphinx gallery
      :link: examples/gallery-examples/index
      :link-type: doc

>       Examples of sphinx gallery rendered using the `ansys-sphinx-theme`.

Reviewer: ```suggestion
      Examples of how the Sphinx gallery is rendered.
```

   .. grid-item-card:: :octicon:`bookmark` Sphinx autoapi
      :link: sphinx-autoapi
      :link-type: ref

>       Examples of autoapi rendered using the `ansys-sphinx-theme`.

Reviewer: ```suggestion
      Examples of how Sphinx AutoAPI documentation is rendered.
```

Documentation components
========================

> These examples show how the different documentation components are rendered in the ``ansys-sphinx-theme``.

Reviewer: ```suggestion
These examples show how the Ansys Sphinx Theme renders different documentation components.
```

.. grid:: 2
   :gutter: 2 2 3 4
   :padding: 2 2 3 4

   .. grid-item-card:: :octicon:`table` Tables
      :link: table
      :link-type: ref

      Examples of tables with JavaScript and RST rendered using the `ansys-sphinx-theme`.

   .. grid-item-card:: :octicon:`info` Admonitions
      :link: admonitions
      :link-type: ref

>       Examples of admonitions rendered using the `ansys-sphinx-theme`.

Reviewer: ```suggestion
      Examples of how admonitions are rendered.
```
